rllib/offline/estimators/doubly_robust.py

```diff
@@ -145,3 +162,61 @@ def train(self, batch: SampleBatchType) -> Dict[str, Any]:
         batch = self.convert_ma_batch_to_sample_batch(batch)
         losses = self.model.train(batch)
         return {"loss": np.mean(losses)}
+
+    @override(OfflineEvaluator)
+    def estimate_on_dataset(self, dataset: Dataset, *, n_parallelism: int = ...) -> Dict[str, Any]:
+
+        dsize = dataset.count()
+        batch_size = max(dsize // n_parallelism, 1)
+        # step 1: clean the dataset and remove the time dimension for bandits
```

Comment:
Should we really auto-infer here that this is used for Bandits only? I feel like the time-dim removal doesn't belong here and should be handled by the client of this class.

Reply:
yeah it doesn't belong here. It now happens in algorithm.py only if the problem is offline bandits